nav_login.php

```diff
@@ -0,0 +1,7 @@
+<!--nav for login page -->
```

Comment:
Make up the login nav for the sign up page that user can access to so that they can make up their account.